Short blogpost about new features (#344)
deltachat/deltachat-pages: _posts/2020-05-06-may-features.md

---
title: Delta Features for May - ECC Keys, Autodelete Messages, Scan QR Codes With Desktop
author: compl4xx
image: ../assets/blog/screenshots/2020-05-06-delta-chat-desktop-scan-qr-code.jpg
---

**While the world was on hold, the Delta Chat development silenty continued.
Here is a short list of features you can try out with the [new

> releases](https://get.delta.chat) coming this May!**

Reviewer: don't like this intro much to be honest.   Would just say "New Delta chat releases! thanks to all the early testers and all the work that went into the new releases on Android, iOS and Desktop. Here is an overview on what you can expect" or something along the lines.  There is enough references to covid19 everywhere, and it's also not really true -- covid19 happenings affected and continue to affect me quite a bit and also others i think ....

Author: Okay, I just took that :) wanted to make it a bit hopeful, but of course I don't know that well what you are going through.

---
title: Delta Features for May - ECC Keys, Autodelete Messages, Scan QR Codes With Desktop
author: compl4xx
image: ../assets/blog/screenshots/2020-05-06-delta-chat-desktop-scan-qr-code.jpg
---

**New Delta chat releases! Thanks to all the early testers and all the work
that went into the new releases on Android, iOS and Desktop. Here is an

> overview on what you can expect:**

Reviewer 2: ```suggestion
overview of what you can expect:**
```

> overview on what you can expect:**

Reviewer 2: ```suggestion
overview of what to expect:**
```

## Desktop 1.3

- Scan QR codes on desktop to verify chats or join groups
- Pin chats to the top if you need them often
- Change the names of your contacts to tell them apart
- For new accounts, ed25519 keys are generated. The new keys are smaller and faster. [This saves traffic](https://delta.chat/en/2020-03-11-reduced-message-size) and time.

![A Screenshot of the Delta Chat Desktop Client, scanning a QR code.](../assets/blog/screenshots/2020-05-06-delta-chat-desktop-scan-qr-code.jpg)

*You can now scan QR codes with the Desktop Client.*

## Android 1.6

- Auto-delete messages from server, and/or from your device
- Have more than one account and switch between them (experimental)
- Search messages in a chat
- Search through the in-app help

> - For new accounts, ed25519 keys are generated. The new keys are smaller and faster. [This saves traffic](https://delta.chat/en/2020-03-11-reduced-message-size) and time.

Reviewer 2: ```suggestion
- For new accounts, Ed25519 keys are generated. The new keys are smaller and faster. [This saves traffic](https://delta.chat/en/2020-03-11-reduced-message-size) and time.
```
As per https://tools.ietf.org/html/rfc8032 and https://tools.ietf.org/html/rfc8080

![Search through chats in the Android App](../assets/blog/screenshots/2020-05-06-delta-chat-android-chat-search.png)

*Search through all messages in a chat, and jump between the found messages.*

## iOS 1.6

- Auto-delete messages from server, and/or from your device
- Get started easier with the new welcome screen
- Create account by scanning a QR code

> - For new accounts, ed25519 keys are generated. The new keys are smaller and faster. [This saves traffic](https://delta.chat/en/2020-03-11-reduced-message-size) and time.

Reviewer 2: ```suggestion
- For new accounts, Ed25519 keys are generated. The new keys are smaller and faster. [This saves traffic](https://delta.chat/en/2020-03-11-reduced-message-size) and time.
```

![Delete Messages Automatically on iOS and Android.](../assets/blog/screenshots/2020-05-06-delta-chat-ios-autodelete.jpg)

> *You can automatically have your messages deleted to save space on your phone or protect yourself against device theft.*

Reviewer 2: ```suggestion
*You can automatically have your messages deleted to save space on your device or protect it against theft.*
```
Identity/information theft?
